platformio.ini

```diff
@@ -296,21 +296,8 @@ lib_deps          = Servo
 
 [env:LPC1769]
 platform          = https://github.com/p3p/pio-nxplpc-arduino-lpc176x/archive/0.1.2.zip
```

Comment:
If you want, you should be able to eliminate this duplication with:

`platform = ${env:LPC1768.platform}`

That way whenever this updates it only has to change in one place.